### DISCUSSION TITLE
How to reset a continuous list?

### DISCUSSION THREAD

**Discussion by danielvartan** [quarto-dev/quarto-cli#7496] [Q&A] [answered]
### Description

Quarto allows listings with interruptions (a continuous list). This can be achieved using (`@`) instead of `1.`.

![image](https://github.com/quarto-dev/quarto-cli/assets/29462472/a9705122-50de-4a93-a0fc-5470fadda35a)

> Source: https://quarto.org/docs/authoring/markdown-basics.html#lists

My question is regarding having more than one continuous list in the same Quarto file `.qmd`. Is there a way to "reset" one of them?

For example:

```
(@)  A list whose numbering

continues after

(@)  an interruption

## New section

(@)  A list whose numbering

continues after

(@)  an interruption
```

In my tests, the numbering just continues without resetting.

**Comment by mcanouil** [comment]
See <https://pandoc.org/MANUAL.html#lists>

**Comment by danielvartan** [comment]
According to the [Pandoc Manual](https://pandoc.org/MANUAL.html#ending-a-list), a `<!-- -->` comment would do the trick.

```md
## Ending a list

What if you want to put an indented code block after a list?
 
-   item one
-   item two

    { my code block }

Trouble! Here pandoc (like other Markdown implementations) will treat { my code block } as the second paragraph of item two, and not as a code block.

To “cut off” the list after item two, you can insert some non-indented content, like an HTML comment, which won’t produce visible output in any format:

-   item one
-   item two

<!-- end of list -->

    { my code block }

You can use the same trick if you want two consecutive lists instead of one big list:

1.  one
2.  two
3.  three

<!-- -->

1.  uno
2.  dos
3.  tres
```

Unfortunately, it doesn't seems to work with Quarto.

```md
### Figures

See [Code Cells](https://quarto.org/docs/reference/cells/).

(@) Create a new figure environment.

Name it ` abntfigure` and include an `\quartolegend` in the end.

Use `\notblank` from `etoolbox` package to add a conditional layer.

(@) Add the new environment to `fig-env` in `_extension.yml`.

<!-- -->
 
### Tables

(@) Add a hook at the end of the `longtable` environment.

Include the `\quartolegend` and utilize `\notblank` from `etoolbox` package to add a conditional layer.
```

Render:

![image](https://github.com/quarto-dev/quarto-cli/assets/29462472/63f0cc45-1709-4272-b424-ff9b9b9e06b0)

**Reply by cderv** [reply]
Here your question is about using a specific list which [example list](https://pandoc.org/MANUAL.html#extension-example_lists) and the Manual says: 
> The first list item with a @ marker will be numbered ‘1’, the next ‘2’, and so on, throughout the document. The numbered examples need not occur in a single list; each new list using @ will take up where the last stopped.

I don't think those can be ended or reset. You can try your code with Pandoc only and see that it will indeed continue. 
This should be asked on Pandoc community directly like https://groups.google.com/g/pandoc-discuss/

The part about https://pandoc.org/MANUAL.html#ending-a-list is impacted other type of list where continuation can be stopped. I believe.

**Comment by adamiturabi** [comment, marked as answer]
Pandoc has added an option `(1@foo)` to explicitly reset the numbered example list count. More details here:https://github.com/jgm/pandoc/issues/10940#issuecomment-5391381779

**Reply by cderv** [reply]
Thanks for the heads up ! 

This is still in main dev branch of pandoc and will be available in next version. We probably update it for next Quarto 1.11. 

I'll try to update when a prerelease of quarto will have it
